<ui version="4.0" >
 <class>QgsGeomTypeDialogBase</class>
 <widget class="QDialog" name="QgsGeomTypeDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>New Vector Layer</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" colspan="2" >
    <widget class="QLabel" name="mFileFormatLabel" >
     <property name="text" >
      <string>File Format:</string>
     </property>
     <property name="buddy" >
      <cstring>mFileFormatComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2" >
    <widget class="QComboBox" name="mFileFormatComboBox" />
   </item>
   <item row="3" column="1" colspan="2" >
    <widget class="QPushButton" name="mRemoveAttributeButton" >
     <property name="text" >
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>91</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QLabel" name="mAttributeLabel" >
     <property name="text" >
      <string>Attributes:</string>
     </property>
     <property name="buddy" >
      <cstring>mAddAttributeButton</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QPushButton" name="mAddAttributeButton" >
     <property name="text" >
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4" >
    <widget class="QGroupBox" name="buttonGroup1" >
     <property name="title" >
      <string>Type</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QRadioButton" name="mPointRadioButton" >
        <property name="text" >
         <string>Point</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="mLineRadioButton" >
        <property name="text" >
         <string>Line</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QRadioButton" name="mPolygonRadioButton" >
        <property name="text" >
         <string>Polygon</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="4" >
    <widget class="QTreeWidget" name="mAttributeView" >
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Type</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="5" column="0" colspan="4" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>mPointRadioButton</tabstop>
  <tabstop>mLineRadioButton</tabstop>
  <tabstop>mPolygonRadioButton</tabstop>
  <tabstop>mFileFormatComboBox</tabstop>
  <tabstop>mAddAttributeButton</tabstop>
  <tabstop>mRemoveAttributeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
